unicodetools/data/security/dev/data/source/confusables-source.txt

```diff
@@ -5437,4 +5450,26 @@ ABBB; 0473; V8_0; ꮻ => ѳ; CHEROKEE SMALL LETTER WI => CYRILLIC SMALL LETTER F
 1F16E ; C 20E0 	; V11_0 ; CIRCLED C WITH OVERLAID BACKSLASH
 # 1F16F ; 🚹	; V11_0 ; CIRCLED HUMAN FIGURE
 
+# 178-A76 — Section 21 of document L2/24-012
 
+513F ; 儿  # V16.0 ; U+513F ︎➡︎ U+16FF2
+16FF3 ; 兒 # V16.0 ; U+5152 ➡  U+16FF3
+ㄦ ; 儿  # V16.0 ;  U+3126 ㄦ BOPOMOFO LETTER ER ➡ 儿
+
+# 176-A116 — Section 2a of L2/23-164
+
+A7DA ; Λ # V16.0 ; U+A7DA LATIN CAPITAL LETTER LAMBDA ➡ greek equiv
+A7DB ; λ # V16.0 ; U+A7DB LATIN SMALL LETTER LAMBDA ➡ greek equiv
+A7DC ; Λ̷̷̷ # V16.0 ; U+A7DC LATIN CAPITAL LETTER LAMBDA WITH STROKE ➡ greek equiv
+ƛ ; λ̷̷̷ # V16.0 ; existing Latin variant
+
+# 165-A37 — L2/20-272
+
+1715 ; 1734 # V16.0 ; U+1715 TAGALOG SIGN PAMUDPOD ➡ 1734, Hanunoo Sign Pamudpod
+
+# 166-A55  — 
+
+ß ; β # sharp S with beta
+ẞ ~ ß # sharp S upper/lower
+A7D6 ; β # Middle Scots S, uppercase
+A7D6 ; β # Middle Scots S, lowercase
```

Comment:
Are you mapping to the beta instead of the sharp s because they are in the same class?

Reply:
Yes, it doesn't matter really. But when I looked at the proposal, the Scots S had more of a tail on it than a sharp S does.